src/utils/worklets/chat/messenger/messages.cljs

```diff
@@ -5,3 +5,9 @@
 (defn messages-list-on-scroll
   [distance-from-list-top chat-list-scroll-y callback]
   (.messagesListOnScroll ^js worklets distance-from-list-top chat-list-scroll-y callback))
+
+(defn use-messages-scrolled-to-top
+  "Returns true if `distance-from-list-top` (animated value) crossed the threshold.
+  e.g. reaching the very top would need a threshold of `0`"
```

Comment:
Or we can use just another name for this? Maybe something like, `.....-scrolled-to-threshold`

Reply:
yeah that's a good point :+1: